src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -52,17 +52,17 @@ final case class RequestSession[F[_]: Monad](
   def flatMap[R, R1](r: F[R], f: R => F[R1]): F[R1] = r.flatMap(f)
 }
 
-class GenericClient[F[_]: Monad, _](applicationName: String)(
+class GenericClient[F[_]: Monad, _](applicationName: String, baseUri: String)(
```

Comment:
The other way to handle this would be to use an implicit parameter, as we do for `Auth`. Then we can have a default implicit that attempts to read from the `COGNITE_BASE_URL` as in https://cognite-docs.readthedocs-hosted.com/projects/cognite-sdk-python/en/latest/cognite.html?highlight=cognite_base_url#environment-configuration

This is a bit more advanced, so I can explain it tomorrow. I think it will be a better user experience, but not totally sure.